patch/gfx9/llpcNggPrimShader.cpp

```diff
@@ -3032,6 +3032,7 @@ void NggPrimShader::RunEsOrEsVariant(
 
     auto pEsArgBegin = pEsEntry->arg_begin();
     const uint32_t esArgCount = pEsEntry->arg_size();
+    LLPC_UNUSED(esArgCount);
```

Comment:
I think we can replace esArgCount with pEsEntry->arg_size(). This variable is no longer actually used.
The same case is gsArgCount in NggPrimShader::RunGsVariant().
Sorry for that. I forget to clear them.